foundation/foundation-configuration-store/src/main/java/com/boozallen/aissemble/configuration/service/ConfigService.java

```diff
@@ -0,0 +1,80 @@
+package com.boozallen.aissemble.configuration.service;
+
+/*-
+ * #%L
+ * aiSSEMBLE::Foundation::Configuration::Store
+ * %%
+ * Copyright (C) 2021 Booz Allen
+ * %%
+ * This software package is licensed under the Booz Allen Public License. All Rights Reserved.
+ * #L%
+ */
+
+import com.boozallen.aissemble.configuration.data.LocationUriInput;
+import com.boozallen.aissemble.configuration.store.ConfigLoader;
+import com.boozallen.aissemble.configuration.store.Property;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.json.JsonMapper;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.inject.Inject;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import java.util.Set;
+
+@Path("/aissemble-properties")
+public class ConfigService {
+    public static final String PROPERTIES_LOADED = "properties are loaded";
+    public static final String ERROR_LOADING_PROPERTIES = "error loading properties.";
+    private static final Logger logger = LoggerFactory.getLogger(ConfigService.class);
+    private static final ObjectMapper mapper = JsonMapper.builder().build();
+
+    @Inject
+    public ConfigLoader configLoader;
+
+    @GET
+    @Path("/{groupName}/{propertyName}")
+    @Produces(MediaType.APPLICATION_JSON)
+    public String getProperty(@PathParam("groupName") String groupName,
+                              @PathParam("propertyName") String propertyName) {
+        Property property = null;
+        try {
+             property = this.configLoader.read(groupName, propertyName);
+        } catch (Exception e) {
+            logger.error("Error reading property: " + e);
```

Comment:
S: Would change to this to make sure the exception is printed properly:
```suggestion
            logger.error("Error reading property: ", e);
```
Also might consider printing group/property that failed to read.

Reply:
Actually, in the same vein as my other comment, it might be nice to just rethrow the exception if there is no security risk in that and/or return a 404.

Reply:
throw exception would result a 500 and with an error id, return a 404 probably makes more sense